src/data/finset/prod.lean

```diff
@@ -71,6 +71,10 @@ product_subset_product hs (subset.refl _)
 lemma product_subset_product_right (ht : t ⊆ t') : s ×ˢ t ⊆ s ×ˢ t' :=
 product_subset_product (subset.refl _) ht
 
+@[simp] lemma image_swap_product [decidable_eq α] [decidable_eq β] (s : finset α) (t : finset β) :
+  (t ×ˢ s).image prod.swap = s ×ˢ t :=
+coe_injective $ by { push_cast, exact set.image_swap_prod _ _ }
```

Comment:
Can you add the version with `map` and `equiv.prod_comm.to_embedding` (and without `decidable_eq`) too?